### ISSUE TITLE
Razer Basilisk V3 not working

### ISSUE THREAD

**Issue by Jayce052** [1kc/razer-macos#557]
Hi does the latest version support the Basilisk V3?

**Comment by WrathofthePast** [comment]
No, but the next version should. The backend work is already merged. We're just waiting on the frontend. 
Duplicate of #464

**Comment by Jayce052** [comment]
Ok thanks so much!

**Comment by JoanaRoqueOS** [comment]
Hi guys. any changes about the supporting of the basilisk v3? just bought it and discover the incompatibility with macOS... I have a macbook pro from 2018 with macOS Monterey versions 12.2.1. 

I install the app, but the mouse is not recognized... i already did the 'refresh device list' and nothing... and then i discover this 'issue' was open... Any news about when you will be able to got it to work with the app??? 
I'm really appreciate your hard work guys!

**Comment by WrathofthePast** [comment]
@Jayce052, can we close this issue in leu of #464?
